### PR TITLE
Add personal userspace, update keymaps

### PR DESCRIPTION
**Pull request by vomindoraan**
Move common features of my keymaps into a userspace (finally).
Change how shared Fn and Numpad layers work.

## Types of changes
- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/konstantin/rules.mk [hunk 1]
```diff
@@ -0,0 +1,16 @@
+BOOTMAGIC_ENABLE   = no
```
Comment (mechmerlin) [inline users/konstantin/rules.mk]: no? Not even lite?

Comment (vomindoraan) [reply users/konstantin/rules.mk]: I really dislike the Bootmagic user experience, Command is a great alternative for me 😃

Comment (vomindoraan) [reply users/konstantin/rules.mk]: Also with lite I'm kind of scared of accidentally putting the board into bootloader mode when holding it and plugging it in.

**Comment by mechmerlin** [comment]
LGTM

**Pull request merged**